### DISCUSSION TITLE
How to run an Ethereum Node?

### DISCUSSION THREAD

**Discussion by ropats16** [Cyfrin/foundry-full-course-cu#1255] [Q&A]
The Blockchain Basics Lesson 10 Timestamp ~2:30 mentions the guide to run an Ethereum Node in seconds linked in the GitHub however unable to find the same.

Is there a resource I can refer to and what are the hardware requirements?

**Comment by ghost** [comment]
Definitely a good question for [r/ethstaker](https://www.reddit.com/r/ethstaker/). If one day I want to run a node that would be my first place to look for help or information. They also have a discord and I'm sure they have a detailed guide on how to setup a node (hardware and software).

**Reply by ropats16** [reply]
Thanks will look into it
